CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing 
```

Comment:
also good practice is to include `.github` templates for creating PR and Issues, see https://github.com/wc-catalogue/blaze-elements/tree/master/.github as a reference